### PR TITLE
Spelling needs upgradefrom

### PR DESCRIPTION
**Pull request by jsoref**
Split from #5709 

I'm leaving this as a todo. I don't think I'm going to do it.

Notes:
* https://github.com/zaproxy/zaproxy/pull/5709#discussion_r351295130

image

### CODE REVIEW

**Comment by kingthorin** [comment]
Remove "component" commit.

**Review by psiinon — CHANGES_REQUESTED** [review]
Looks good but needs a ZAP comment at the top.

**Comment by kingthorin** [comment]
Fixed

**Review by psiinon — APPROVED** [review]

File: zap/src/main/java/org/parosproxy/paros/Constant.java [hunk 1]
```diff
@@ -1113,6 +1119,21 @@ private static void upgradeFrom2_8_0(XMLConfiguration config) {
         updatePscanTagMailtoPattern(config);
     }
 
+    private static void upgradeFrom2_9_0(XMLConfiguration config) {
+        String old_key_name = ".markocurrences"; // This is the typo we want to fix
+        String new_key_name = ".markoccurrences";
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply zap/src/main/java/org/parosproxy/paros/Constant.java:1124]: Oh sure, I hadn't even though of checking for unittests.

**Comment by kingthorin** [comment]
Done & done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**